src/kemal.cr

```diff
@@ -69,15 +69,12 @@ module Kemal
   end
 
   def self.stop
-    if config.running
-      if server = config.server
-        server.close unless server.closed?
-        config.running = false
-      else
-        raise "Kemal.config.server is not set. Please use Kemal.run to set the server."
-      end
+    raise "Kemal is already stopped." if !config.running
```

Comment:
`unless` reads better than `if !`

Reply:
That's an opinion, for simple single condition I personally prefer a short `if !` which means "if different".

Reply:
Yeah, it might be a question of opinion, to prefer one or the other. But it should be consistent in the code base and not switch between both every other line.